controllers/drplacementcontrol.go

```diff
@@ -2223,6 +2221,9 @@ func (d *DRPCInstance) fixupPlacementForFailover() error {
 		primary, secondaries = d.selectCurrentPrimaryAndSecondaries()
 	}
 
+	d.log.Info(fmt.Sprintf("Fixing PlacementRule for failover. Primary (%s), Secondaries (%v)",
+		primary, secondaries))
```

Comment:
This is a third type of change that not related to log levels, so it should be a separate commit (show more context in logs) with similar changes.

Reply:
Removed